### PR TITLE
Adding tslint configuration

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?

Adding tslint configuration.

### CODE REVIEW

**Comment by evidolob** [comment]
Why you add tslint configuration in to the root `package.json`?
Maybe it's better to add to [theia-dashboard-extension/package.json](https://github.com/eclipse/che-theia-dashboard-extension/blob/master/theia-dashboard-extension/package.json)?

**Comment by sunix** [comment]
@evidolob to me it is fine. easier to find when it is in the root folder.
and usually use yarn from the root folder as well. So invoking it from here is ok and more convenient.

File: package.json [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
-    "prepare": "lerna run prepare",
-    "rebuild:browser": "theia rebuild:browser"
+    "prepare": "yarn tslint && lerna run prepare",
+    "rebuild:browser": "theia rebuild:browser",
+    "tslint-fix": "tslint --fix --project theia-dashboard-extension",
+    "tslint": "tslint --project theia-dashboard-extension"
   },
+  "license": "EPL-2.0",
   "devDependencies": {
-    "typescript": "2.7.2",
```
Comment (sunix) [inline package.json:8]: could you explain why we remove this dependency ? is it related to tslint config ?

Comment (olexii4) [reply package.json:8]: It is a line with code clean up. We have needed version in [theia-dashboard-extension/package.json](https://github.com/eclipse/che-theia-dashboard-extension/blob/master/theia-dashboard-extension/package.json#L17). I just forgot about it after some experiments with versions.

**Comment by olexii4** [comment]
@evidolob I added tslint configuration into the root package.json to be in a common way as we already have in [che-theia-machines-plugin/package.json](https://github.com/eclipse/che-theia-machines-plugin/blob/master/package.json)

File: package.json (hunk 1 above)
Comment (sunix) [reply package.json:8]: ok, you should make it a separate commit or PR, as you wish.

Comment (olexii4) [reply package.json:8]: done

**Review by sunix — APPROVED** [review]
LGTM

**Review by evidolob — APPROVED** [review]

**Pull request merged**